qcodes/tests/conftest.py

```diff
@@ -51,7 +51,7 @@ def disable_telemetry():
         qc.config.telemetry.enabled = original_state
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
@jenshnielsen maybe the default_config fixture adds a lot of overhead if applied to every test. The reason it is so complicated is that it takes into account that the config might be reloaded somewhere. Maybe this is the time to go through the use of the config and make sure it is not dynamically reloaded, or make the reloading easier. In that case we could reduce this to:

```python
   current_config, current_schema = qc_config.current_config, qc_config.current_schema
    try:
        qc_config.current_config, qc_config.current_schema = qc_config.load_default()
        yield
    finally:
        qc_config.current_config, qc_config.current_schema = current_config, current_schema

```

Reply:
@Dominik-Vogel reworked this please have a look at the new version